gradle/kotlin.gradle

```diff
@@ -18,16 +18,6 @@ apply plugin: "nebula.kotlin"
 
 compileKotlin {
   kotlinOptions {
-    languageVersion = "1.2"
-    jvmTarget = "1.8"
-  }
-}
-configurations.all {
-  resolutionStrategy {
-    eachDependency { details ->
-      if (details.requested.group == "org.jetbrains.kotlin") {
-        details.useVersion kotlinVersion
-      }
-    }
```

Comment:
Kotlin's bom (which we get transitively from `kork-bom`) does this alignment now.